concent_api/core/admin.py

```diff
@@ -50,12 +71,17 @@ class SubtaskAdmin(ModelAdminReadOnlyMixin, admin.ModelAdmin):
 
     def get_provider_public_key(self, obj):  # pylint: disable=no-self-use
         return obj.provider.public_key
+
```

Comment:
There is no need for this extra lines, in Django admin convention it is ok. Is pylint complaining about it ?

Reply:
Done.